### ISSUE TITLE
Cannot load more than one proto file

### ISSUE THREAD

**Issue by ecbaldwin** [ktr0731/evans#80]
I have a need to load more than one proto file because the definitions I'm loading are not contained in one file. I happen to be trying to load the proto files from [gobgp][1]. This is what I get.

[1]: https://github.com/osrg/gobgp/tree/3dc23d3075f13f2976afd255ef2fd9b1410acc77/api

```
# evans api/*.proto
1 error occurred:

* gobgp.proto:292:3: duplicate symbol gobgpapi.AddBmpRequest.port: already defined as field in "api/gobgp.proto"
```

These  symbols are not defined twice. Any ideas?

```
# evans --version
evans 0.5.1
```

**Comment by ecbaldwin** [comment]
Okay, I figure out what happened. I gave three file on the command line: `api/gobgp.proto`, `api/attribute.proto`, and `api/capability.proto`. Evans code [puts the directory part on the import path][1] for protoreflect. Then it passes the filenames, unmodified, to it.

While recursively reading the files and their imports, it finds that `api/capability.proto` [imports `gobgp.proto`][2]. It does not tell that it is the same file as `api/gobgp.proto`. Arguably, it shouldn't try to figure that out since proto files can be imported using a relative hierarchy. It tries to read it again and it finds everything is duplicated.

[1]: https://github.com/ktr0731/evans/blob/10a53769b9d6469b725b7015e8959461a8df3198/adapter/internal/protoparser/parser.go#L18-L24
[2]: https://github.com/osrg/gobgp/blob/3dc23d3075f13f2976afd255ef2fd9b1410acc77/api/capability.proto#L24

**Comment by ecbaldwin** [comment]
#81, which I just posted, is a short fix to the problem. However, I'm wondering if it is the correct fix. I wonder about the wisdom of putting the directory part of each file on the import path. Files can be imported with a hierarchical path, which would be resolved relative to the path. Hence, there should be a semantic difference between

```
cd api && evans gobgp.proto attribute.proto capability.proto
```

and

```
evans api/gobgp.proto api/attribute.proto api/capability.proto
```

Now that I think about it, the second one should fail because those protos are not written to be imported with hierarchical paths. I think the better fix would be to add only `.` to the import path and not do all of the magic of trying to guess that the directory part of the filename should be added to the path. I'll post another PR.

**Comment by ktr0731** [comment]
fixed at #82
